docs/azure_jumpstart_arcbox/Full/_index.md

```diff
@@ -530,6 +530,50 @@ Open the [data services operations page](https://azurearcjumpstart.io/azure_jump
 
   ![Screenshot showing Grafana dashboard](./activity1.png)
 
+### Arc-enabled SQL Server - Best practices assessment
+
+As part of the ArcBox deployment SQL Server best practices assessment is configured and ran SQL assessment to show assessment results. Open ArcBox-SQL Arc-enabled SQL Server resource from the resource group deployed or Azure Arc service blade to view SQL Serve best practice assessment results.
+
+- Following screenshot shows the SQL Server best practices assessment page and the scheduled and previously ran assessments. If this page do not show assessment results click on Refresh button show assessments. Once displayed the assessments and results click on _View assessment_ results to see results.
+
+  ![Screenshot showing SQL Server best practices assessment configuration](./sql-pba-view-results.png)
+
+  ![Screenshot showing SQL Server best practices assessment results part 1](./sql-bpa-results-1.png)
+
+  ![Screenshot showing SQL Server best practices assessment results part 2](./sql-bpa-results-2.png)
+
+### Microsoft Defender for Cloud - SQL servers on machines
+
+This section guides you through different settings for enabling Microsoft Defender for Cloud - SQL servers on machines. Most of these settings are already enabled during the logon script execution when login to _ArcBox-Client_ Azure VM. Even though these are pre-configured there might be delays in showing them in the Azure portal.
+
+- Following are the settings of Microsoft Defender for Cloud - SQL servers on machines configured using automation scripts and can be reviewed in Azure portal.
+
+  ![Screenshot showing Microsoft Defender for Cloud plans](./microsoft-defender-plans.png)
+
+  ![Screenshot showing Microsoft Defender for Cloud SQL enabled](./defender-sql-plan.png)
+
+- The below screenshots show Arc-enabled SQL Server Defender for Cloud enablement and protection status. Defender for Cloud for SQL Server is enabled at the subscription level, but the protection status is still showing as not enabled.
+Please note it may take some time to show this status in the Azure portal, but still able to detect SQL threats generated by the test scripts.
```

Comment:
```suggestion
- The below screenshots show Arc-enabled SQL Server Defender for Cloud enablement and protection status. Defender for Cloud for SQL Server is enabled at the subscription level, but the protection status is still showing as not enabled.

Please note it may take some time to show this status in the Azure portal, but still able to detect SQL threats generated by the test scripts.
```